experiments/snippets/skuidCustom/skuidCustom.js

```diff
@@ -0,0 +1,1344 @@
+				}
+
+				if (encloseValueInQuotes === true) {
+					valuePre = '\'';
+					valuePost = '\'';
+				}
+
+				having += `(${fieldfunction}${fieldPre}${field}${fieldPost} ${operator} ${valuePre}${value}${valuePost})`;
+			}
+
+			if (having !== '') {
+				having += ')';
+			}
+		}
+
+		const queryStr = `SELECT ${fieldsStr} FROM ${model.objectName}${ourConditionStrPre}${ourConditionStr}${lastRowConditionsPre}${lastRowConditions}${groupbyPre}${groupby}${havingPre}${having}${orderby} LIMIT ${limit}`;
+
+		// console.log('QUERY:');
+		// console.log(queryStr);
+
+		skuid.$.when(skuid.sfdc.api.query(queryStr)).done(function (queryResult) {
+			// Records is an array of query results in the format {field: value, field2: value2}
+			const records = queryResult.records;
+
+			if (records !== undefined && records !== null && records.length > 0) {
+				for (let i = 0; i < records.length; i++) {
+					// Remove attributes, we won't adopt that field into our rows
+					if (records[i].attributes !== undefined) {
+						delete records[i].attributes;
+					}
+				}
+
+				// Adopt our rows into our tempModel
+				tempRows = tempRows.concat(records);
+			}
+			
+			if (records.length === limit) {
+				// If we hit our query limit, we need to query again
+				if (progressCallback !== undefined) {
+					progressCallback.call(this, {
+						count,
+						limit,
+						nextStart: count + records.length + 1,
+						nextEnd: count + (records.length * 2)
+					});
+				}
+
+				modelLoadSOQL(model, {
+					limit,
+					last: records[limit - 1],
+					count: count + records.length,
+					progressCallback,
+					promiseResolve,
+					promiseReject,
+					tempRows,
+					exportWhenDone,
+					exportOptions
+				});
+			} else {
+				// Otherwise we've finished querying, resolve
+				// sort our tempRows based on original sorting parameters set in the model
+				tempRows = sortRows(tempRows, model);
+
+				// adopt our tempRows into our model
+				model.adoptRows(tempRows);
+
+				if (exportWhenDone) {
+					skuid.$.when(model.exportData(exportOptions)).done((result) => {
+						promiseResolve(result);
+					}).fail((result) => {
+						promiseReject(result.error);
+					});
+				} else {
+					promiseResolve();
+				}
+			}
+		}).fail((queryResult) => {
+			promiseReject(queryResult.error);
+		});
+
+		function getAggFieldIdFromName (model, name) {
+			if (model.isAggregate === false) {
+				return name;
+			}
+
+			for (let i = 0; i < model.fields.length; i++) {
+				if (model.fields[i].function === undefined && model.fields[i].name === name) {
+					return model.fields[i].id;
+				}
+			}
+
+			return name;
+		}
+
+		function getAggFieldNameFromId (model, id) {
+			if (model.isAggregate === false) {
+				return id;
+			}
+
+			for (let i = 0; i < model.fields.length; i++) {
+				if (model.fields[i].function === undefined && model.fields[i].id === id) {
+					return model.fields[i].name;
+				}
+			}
+
+			return id;
+		}
+
+		function operatorConvert (operator) {

```

Comment:
this function already exists elsewhere in this file: you could use `skuid.custom.operatorSOQL` here rather than redefining the function

Reply:
removed the local operatorConvert functions, using only skuid.custom.operatorSOQL now